content/en/docs/instrumentation/python/automatic.md

```diff
@@ -88,6 +92,15 @@ The examples that follow send instrumentation results to the console. Learn more
 about installing and configuring the [OpenTelemetry Distro](../distro) to send
 telemetry to other destinations, like an OpenTelemetry Collector.
 
+**Note**: If you were to use auto-instrumentation and manually instrumenting your code the
```

Comment:
Suggested rewrite for ln 95-102:

> _Note_: If utilizing automatic instrumentation through `opentelemetry-instrument`, you must configure it via environment variables or the command line. The agent creates a telemetry pipeline that cannot be modified other than through these means. If you need more customization for your telemetry pipelines, then you may opt to forego the agent and import the OpenTelemetry SDK and instrumentation libraries into your code and configure them there. You may also extend automatic instrumentation by importing the OpenTelemetry API. Please review the [API documentation](https://opentelemetry-python.readthedocs.io/en/latest/index.html) for more details.

Reply:
@austinlparker - the start of the note doesn't make sense. Should it read "To use" instead of "If utilizing"?

Reply:
sgtm

Reply:
> Suggested rewrite for ln 95-102:

Done, with some copyedit tweaks.